lib/td/command/common.rb

```diff
@@ -247,6 +247,17 @@ def get_table(client, db_name, table_name)
     table
   end
 
+  def exist_table?(client, db_name, table_name)
```

Comment:
But here's the code to be reused: https://github.com/treasure-data/td/blob/master/lib/td/command/table.rb#L407-L429